### ISSUE TITLE
Not able to use all sprig functions

### ISSUE THREAD

**Issue by elyezer** [tazjin/kontemplate#152]
In the docs https://github.com/tazjin/kontemplate/blob/master/docs/templates.md it mentions being able to call any sprig function but when trying to use the ternary function http://masterminds.github.io/sprig/defaults.html#ternary it raises the following error:

```
kontemplate: error: Error templating resource sets: Could not load template ocp/cloudigrade/build-configs.yaml: template: build-configs.yaml:1: function "ternary" not defined
```

**Comment by tazjin** [comment]
This function may be from a newer version of sprig than what is [currently pinned](https://github.com/tazjin/kontemplate/blob/master/deps.nix#L12-#L20) in Kontemplate. I'm actually way overdue with a new release, once I get to it I'll include a version bump, too.

**Comment by elyezer** [comment]
Let me know if I can help with something.

Thank you

**Comment by tazjin** [comment]
A dependency bump is included in [version 1.7.0](https://github.com/tazjin/kontemplate/releases/tag/v1.7.0). This should solve your issue!

**Comment by elyezer** [comment]
Thank you very much!
